shared/lib/e2e-test-manager.ts

```diff
@@ -58,6 +58,26 @@ export class E2eTestManager {
     return createUser(this.dataSource, additionalData);
   }
 
+  async deleteUserById(id: string) {
```

Comment:
In our context, overall this is a good idea. But let me share some experience that might help you in the future. Right now we don't have a lot of different entities/resources in our system, so a full cleanup is manageable, and for specific use cases this is the best approach.

But soon enough this can grow into a 10 line afterEach hook where you need to delete 9 resources out of 10 from the DB. For these contexts, a:

```
const entitiesToSurviveCleanUp = [Sections, BaseWidgets]
await clearDatabase( { except: entitiesToSurviceCleanUp } )
```

Might be a lot more helpful and clean. No need to implement this, just wanted to point it out for your future mental health :)